src/core_modules/capture-core/metaData/RenderFoundation/CustomForm.js

```diff
@@ -34,13 +34,17 @@ export class CustomForm {
      *
      * @memberof CustomForm
      */
-    set data(html: string) {
-        const data = parseHtml(html, {
-            onTransform: this.transformNode,
+    setData(html: string, transformFunction: () => ?React.Element<'FormField'>) {
+        this._data = parseHtml(html, {
+            onTransform: transformFunction,
             allowScript: true,
         });
+    }
+
+    set data(data: Object) {
         this._data = data;
     }
+
     get data(): Data {
         return this._data;
     }
```

Comment:
I think we can safely remove these?

Reply:
The program will throw an error if we remove the `get data()`, but no errors on removing `set Data()`.
Is it okay if we keep `get data()`, or should I try to find a fix?

Reply:
Good point 👍 Yep, let's keep `get data()` and remove the setter.